programs/client/Client.cpp

```diff
@@ -178,6 +178,9 @@ void Client::parseConnectionsCredentials(Poco::Util::AbstractConfiguration & con
                 history_file = home_path + "/" + history_file.substr(1);
             config.setString("history_file", history_file);
         }
+        if (config.has(prefix + "accept-invalid-certificate")) {
+            config.setString("accept-invalid-certificate", prefix + "accept-invalid-certificate");
+        }
```

Comment:
...I don't understand what this suppose to do?

Reply:
it should handle `accept-invalid-certificate` on a per-connection basis.